readme.md

```diff
@@ -63,14 +63,15 @@ Hello, Node!
 * **Question:** Describe the difference between these two methods.
 
 * **Your Answer:**
+writeFile writes the entire file and replaces existing file if there is one appendFile adds to a file, does not overwrite an existing file.
 
 ---
 
 - [ ] Imagine you want to edit the middle of a file. You can use the [fs module](https://nodejs.org/api/fs.html) and the JavaScript language.
 
 * **Question:** How would you do so?
 
-* **Your Answer:**
+* **Your Answer:**  Not quite sure I understand the question or the scenario. If it was randomly in the middle, I would take the string object return from reading the file and then split that string and add the new data in that position. 
```

Comment:
It's similar to what you are doing in your code. The basic premise is:
1. Read in the file
2. Convert the content to a workable format, if necessary (e.g. `JSON.parse()`)
3. Make edits to the content
4. Reformat the content to the original file format, if necessary (e.g. `JSON.stringify()`)
5. Write the content back to the file